api/Controllers/HotelsController.cs

```diff
@@ -85,8 +93,34 @@ public async Task<IActionResult> GetHotelCards([FromQuery] PageParameters pagePa
 
         [Route("count")]
         [HttpGet]
-        public async Task<IActionResult> GetHotelsCount() =>
-            Ok(await _hotelsDb.Hotels.CountAsync());
+        public async Task<IActionResult> GetHotelsCount([FromQuery] FilterParameters filterParams)
```

Comment:
Do you really need separate endpoint for this?

Reply:
I think I should have this endpoint. In REST each resource should have it's own url, so that endpoint is hotel number url. Will be glad to discuss this point if I'm wrong